src/Operators/spectralelement.jl

```diff
@@ -1355,7 +1405,7 @@ function apply_operator(op::WeakCurl{(1, 2)}, space, slabidx, arg)
     else
         error("invalid return type")
     end
-    for j in 1:Nq, i in 1:Nq
+    @inbounds for j in 1:Nq, i in 1:Nq
```

Comment:
you don't need `@inbounds` here if you use `@propagate_inbounds`

Reply:
Ah, interesting, I thought `@inbounds` was needed any time a new index was introduced. Good catch!